blazor/rating/appearance.md

```diff
@@ -95,16 +95,11 @@ You can change the rating item border color in Blazor Rating component, you can
 
 ![Blazor Rating Component with rating symbol border color](images/blazor-rating-border-color.png)
 
-### Change rated symbol fill color 
+### Changing rated / un-rated symbol fill color 
 
-You can customize the rated fill color of the icons in the Rating control by using the `CssClass` property and setting the **first** color stop of the `linear-gradient` to define the rated fill color using the `background` CSS property of `.e-rating-icon`.
+You can use `CssClass` property to customize the rated / un-rated fill color of the icons, by set the **first** color stop of the `linear-gradient` to define the rated fill color, and the **second** color stop to define the un-rated fill color using the `background` CSS property of `.e-rating-icon`. 
```

Comment:
You can customize the fill colors of rated and un-rated icons in the Rating control using the CssClass property and the linear-gradient color stops in the background CSS property of .e-rating-icon. The first color stop defines the rated fill color and the second defines the un-rated fill color.

Reply:
updated the content as per comments.